ui/raidboss/data/06-ew/raid/p10n.ts

```diff
@@ -1,14 +1,201 @@
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  silkspitOnYou: boolean;
+}
+
+const bossNameUnicode = 'Pand\u00e6monium';
 
 const triggerSet: TriggerSet<Data> = {
   id: 'AnabaseiosTheTenthCircle',
   zoneId: ZoneId.AnabaseiosTheTenthCircle,
   timelineFile: 'p10n.txt',
-  triggers: [],
+  initData: () => {
+    return {
+      silkspitOnYou: false,
+    };
+  },
+  triggers: [
+    {
+      id: 'P10N Silkspit You',
+      type: 'HeadMarker',
+      netRegex: { id: '01CE' },
+      condition: (data, matches) => data.me === matches.target,
+      alertText: (data, _matches, output) => {
+        data.silkspitOnYou = true;
+        return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Away from party and posts',
```

Comment:
Definitely agree with "away from posts", but isn't it "spread from other webs and posts" (wording not great) but trying to differentiate that the goal (I think, but am really not sure) is that the marked people spread from each other and posts, and the party then spreads from the marked people.

Reply:
I struggled a bit with wording on this one too.  If you have the marker, you can tether to (a) other marked players (fairly decent length tethers), (b) any unmarked players (close proximity), or (c) any of the ground posts (somewhere in between distance-wise).  Maybe just "spread and do not tether"?

Reply:
I do think the "away from posts" is good though, as it's not as obvious (at least it wasn't to me).  "Spread from webs and posts", or maybe "Spread (avoid posts)"?

Reply:
I'll toss my vote in for `Spread (avoid posts)` as well. I think it's the best compromise on length while ensuring that anyone reading it will Do The Thing in some way.